stablehlo/reference/Element.h

```diff
@@ -69,6 +69,9 @@ class Element {
   /// Overloaded - operator.
   Element operator-(const Element &other) const;
 
+  /// Overloaded * operator.
+  Element operator*(const Element &other) const;
+
```

Comment:
Let's keep operators ordered alphabetically. This would mean that `*` goes before `+`.

Reply:
Done, but there are multiple ways depending what you call the operator
multiply -> plus -> subtract
or add -> multiply -> subtract

add -> multiply -> subtract
this seems the best choice to me (it matches the op names in spec and op definitions)